### ISSUE TITLE
Missing textLink colors

### ISSUE THREAD

**Issue by mserajnik** [arcticicestudio/nord-visual-studio-code#154]
I felt like the issue template was a bit overkill for this; please let me know if this format is not acceptable.

---

## Issue

The theme seems to be missing the custom colors for `textLinkForeground` and `textLinkActiveForeground`, causing certain links to use the default color `#3794ff` and look like this:

![image](https://user-images.githubusercontent.com/28833155/63219855-075a3000-c17b-11e9-859a-72f3873a8f71.png)

## Workaround

The default color for [`textLinkForeground`](https://github.com/microsoft/vscode/blob/52ecb5bf560ab517495463b6c392586977a9925b/src/vs/platform/theme/common/colorRegistry.ts#L193) and [`textLinkActiveForeground`](https://github.com/microsoft/vscode/blob/52ecb5bf560ab517495463b6c392586977a9925b/src/vs/platform/theme/common/colorRegistry.ts#L194) is `#3794ff` when using a dark theme.

This is the same as for [`pickerGroupForeground`](https://github.com/microsoft/vscode/blob/52ecb5bf560ab517495463b6c392586977a9925b/src/vs/platform/theme/common/colorRegistry.ts#L248), for which you already [use](https://github.com/arcticicestudio/nord-visual-studio-code/blob/f5c767fdc0d0a2a47cca48addd31c3d7325fa89b/themes/nord-color-theme.json#L176) `#88c0d0`.

According to this, setting the following manually in VS Code's `settings.json` brings the desired result:

```json
{
  "workbench.colorCustomizations": {
    "[Nord]": {
      "textLink.foreground": "#88c0d0",
      "textLink.activeForeground": "#88c0d0"
    }
  }
}
```

**Comment by arcticicestudio** [comment]
Hi @mserajnik :wave:, thanks for your contribution :+1:

It looks like there are even more colors for text related elements that are not yet supported by Nord :hushed: 
I've checked for the missing `textLink.activeForeground` theme key you've found and [tracked down the commit where it was added][c] to VS Code. They've added multiple new theme key in through https://github.com/microsoft/vscode/issues/26298 that was merged over 2 years ago and released for the first time in stable version [1.13.0][] but were never mentioned in the release notes.

I'll add all the currently missing theme keys:

- `textSeparator.*`
- `textBlockQuote.*`
- `textPreformat.*`
- `textLink.*`
- `textCodeBlock.*`

I've also created a small daemon (in Go :blue_heart:) running on my server that checks for new commits in open source projects for specific files and creates reports. I guess I'll add VS Code's `colorRegistry.ts` file and parse it to check for other missing keys that were not documented in any release notes.

Regarding the issue template: Your issue is totally fine and well understandable! The issue template is only so detailed and extensive to prevent one-liner or "title-only" reports as well as collecting as many details as possible to better reproduce problems. When you've got 400+ issues/PRs in your queue you're thankful for any bit of additional information to process this batches :wink: 

[c]: https://github.com/microsoft/vscode/blame/752acd504206f00ffe4ed101c2496540d0197f56/src/vs/platform/theme/common/colorRegistry.ts#L164-L170
[1.13.0]: https://code.visualstudio.com/updates/v1_13
